server/src/org/apache/cloudstack/network/ssl/CertServiceImpl.java

```diff
@@ -14,7 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
-package org.apache.cloudstack.network.lb;
+package org.apache.cloudstack.network.ssl;
```

Comment:
https://github.com/apache/cloudstack/pull/2071/commits/ae4e571d510fa7fae74229c8eb550aead36b866b#diff-ebb55a3f7379d7132877a94057342689R39 shows "org.apache.cloudstack.network.tls". I will have a quick check to see what happened locally?